drizzlepac/haputils/poller_utils.py

```diff
@@ -1017,44 +1197,69 @@ def build_poller_table(input: str, log_level, all_mvm_exposures=[], poller_type=
 
-                input_table['skycell_new'] = [int(not BOOL_STR_DICT[str(val).upper()]) for val in input_table['skycell_new']]
+                input_table["skycell_new"] = [
+                    int(not BOOL_STR_DICT[str(val).upper()])
+                    for val in input_table["skycell_new"]
+                ]
             is_poller_file = True
 
             # Check that each file listed in the poller file exists in the current working directory. If a
             # file is missing, copy it over from the path specified in the poller file. Failing that, raise
             # an exception and exit.
             for table_line in input_table:
-                if os.path.exists(table_line['filename']):
-                    log.info(f"Input image {table_line['filename']} found in current working directory.")
-                elif os.path.exists(table_line['pathname']):
-                    log.info(f"Input image {table_line['filename']} not found in current working directory. However, it was found in the path specified in the poller file.")
-                    shutil.copy(table_line['pathname'], os.getcwd())
-                    log.info(f"Input image {table_line['pathname']} copied to current working directory.")
+                if os.path.exists(table_line["filename"]):
+                    log.info(
+                        f"Input image {table_line['filename']} found in current working directory."
+                    )
+                elif os.path.exists(table_line["pathname"]):
+                    log.info(
+                        f"Input image {table_line['filename']} not found in current working directory. However, it was found in the path specified in the poller file."
+                    )
+                    shutil.copy(table_line["pathname"], os.getcwd())
+                    log.info(
+                        f"Input image {table_line['pathname']} copied to current working directory."
+                    )
                 else:
-                    log.error(f"Input image {table_line['filename']} not found in current working directory.")
-                    log.error(f"Archived input image {table_line['pathname']} not found.")
+                    log.error(
+                        f"Input image {table_line['filename']} not found in current working directory."
+                    )
+                    log.error(
+                        f"Archived input image {table_line['pathname']} not found."
+                    )
                     err_msg = f"Input image {table_line['filename']} missing from current working directory and from the path specified in the poller file. Exiting... "
                     log.error(err_msg)
                     raise Exception(err_msg)
-        
-        elif (poller_type == 'mvm') & (len(input_table.columns) != len(poller_colnames)):
-            log.error(f"MVMs should use full poller files with {len(poller_colnames)} columns.")
+
+        elif (poller_type == "mvm") & (
```

Comment:
This is an old incorrect statement.  This should be a logical **and** versus a bitwise **&**!